### PR TITLE
Update clamav rest config to just take a url

### PR DESCRIPTION
**Pull request by easternbloc**
This makes configuration easier of our clamav rest server

### CODE REVIEW

File: controllers/file.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ function clamAV(req, res, next) {
   };
 
   request.post({
-    url: `http://${config.get('clamRest.host')}:${config.get('clamRest.port')}/scan`,
+    url: `${config.get('clamRest.url')}`,
```
Comment (lennym) [inline controllers/file.js]: Get rid of the no-op template literal.

File: test/file.js [hunk 2]
```diff
@@ -118,7 +115,7 @@ describe('/file', () => {
               if (err) {
                 throw err;
               }
-              assert.ok(res.body.url.indexOf('http://localhost/file/') !== -1);
+              assert.ok(res.body.url.indexOf('https://localhost/file/') !== -1);
```
Comment (lennym) [inline test/file.js]: Not a blocker, but this url isn't set in the test file anywhere, so is testing *current* configuration, which makes it liable to be fragile if config changes in future.

I'd probably want to explicitly set the config value in the test to make it robust.

File: controllers/file.js (hunk 1 above)
Comment (easternbloc) [inline controllers/file.js]: hangover from before will do

File: test/file.js (hunk 2 above)
Comment (easternbloc) [inline test/file.js]: I was just thinking that will add it now

**Comment by easternbloc** [comment]
@lennym updated

**Comment by easternbloc** [comment]
sweet

**Pull request merged**